src/core/server/ui_settings/saved_objects/migrations_790.test.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { migrations790 } from './migrations_790';
+import { SavedObjectUnsanitizedUiSettings } from './types';
+
+describe('ui_settings 7.9.0 migrations', () => {
+  const migration = migrations790['7.9.0'];
+
+  test('returns doc on empty object', () => {
+    expect(migration({} as SavedObjectUnsanitizedUiSettings)).toEqual({
+      references: [],
+    });
+  });
+  test('properly renames siem attributes to securitySolution', () => {
+    const doc = {
+      type: 'config',
+      id: '8.0.0',
+      attributes: {
+        buildNum: 9007199254740991,
+        'siem:defaultAnomalyScore': 59,
+        'siem:enableNewsFeed': false,
```

Comment:
Shouldn't we be searching for `siem:` instead of `siem` in the migration function then?

Reply:
why? we're not replacing the colon

Reply:
To avoid false positive on settings starting with `siem` (`siemSomething`), as your prefix seems to be `siem:` ? That's seems like a precaution without drawbacks?

Reply:
i mean, alright. if they have a setting with siemSomething theyre in trouble for different reasons...

Reply:
Failed upgrades caused by less-than-ideal migrations have been a big pain point for our users. Anything we can do to be defensive in this area is good, even if we don't anticipate a specific problem.